ui/perfherder/alerts/AlertsViewControls.jsx

```diff
@@ -60,6 +61,16 @@ export default class AlertsViewControls extends React.Component {
     const { setFiltersState, updateViewState } = this.props;
     setFiltersState({ status });
     updateViewState({ page: 1 });
+    if (['invalid', 'reassigned', 'downstream'].includes(status)) {
+      this.setState({ disableHideDownstream: true });
+      const { filters } = this.props;
+      const actaulValue = filters.hideDownstream;
+      if (actaulValue) {
+        this.updateFilter('hideDownstream');
+      }
+    } else {
+      this.setState({ disableHideDownstream: false });
```

Comment:
If I change the status from reassigned to untriaged the button stays disabled, it needs to be enabled again.

Reply:
After I refresh the page and I select "reassigned" alerts for example, the status in the dropdown remains selected to untriaged, only the button is disabled.